charts/flipt/templates/ingress.yaml

```
@@ -1,6 +1,6 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "flipt.fullname" . -}}
-{{- $svcPort := default .Values.service.httpPort .Values.service.httpsPort -}}
+{{- $svcPort := .Values.ingress.servicePort -}}
```

revert this change and make it 

{{- $svcPort := default .Values.service.httpsPort .Values.service.httpPort -}}

---

One non-breaking alternative between these two that we could do:

```suggestion
{{- $svcPort := default .Values.ingress.servicePort (default .Values.service.httpsPort .Values.service.httpPort) -}}
```

This would preserve the existing behaviour incase anyone is depending on it.
Then it introduces the new lever in this PR for overriding those via `servicePort`.
Thoughts?

---

Good point.